webpack/actions/response_set_actions.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import routes from '../routes';
+import {
+  FETCH_RESPONSE_SETS
+} from './types';
+
+export function fetchResponseSets() {
+  return {
+    type: FETCH_RESPONSE_SETS,
+    payload: axios.get(routes.responseSetsPath(), {
+      headers: {'Accept': 'application/json'}
```

Comment:
Add `'X-Key-Inflection': 'camel'` to the headers so the JSON comes back in camelCase format.